### ISSUE TITLE
Add MAXO Annotation ingest

### ISSUE THREAD

**Issue by kevinschaper** [monarch-initiative/monarch-app#798]
Create an initial MAXO annotation ingest. 

Process is started here: https://github.com/monarch-initiative/maxo-annotation-ingest

This one felt natural to start with doc, then move into tests, then implementation

A draft PR for adding medical action entity & association classes to the biolink model https://github.com/biolink/biolink-model/pull/1523

This may go into the September release using high level classes like `Association` and `NamedThing` 

- [X]  download yaml 
- [x]  transform python
- [x]  transform yaml
- [X]  tests
- [X]  documentation
- [x] import into monarch-ingest
- [ ] Show maxo annotations on monarch-app

**Comment by kevinschaper** [comment]
@sierra-moxon I'm getting stuck on how to capture a CHEBI `extension id` of a MAXO term. 

If the MAXO term is `calcium channel blocking agent therapy` (MAXO:0000434) and the CHEBI term is `2,4-diaminopyridine` (CHEBI:51599), I'm guessing that's saying that this particular chemical is used for the therapy? So then, this is a qualifier that is making the subject more specific? 

maybe subject_derivative_qualifier, because this is a post-composition that means `calcium channel blocking agent therapy with 2,4-diaminopyridine`? 

@iimpulse does that meaning sound right from your end?

**Comment by iimpulse** [comment]
Yeah sounds right. Sometimes it can be less specific though. Drug Therapy and then some specific drug. But I dont think that changes anything here. Just something to think about

**Comment by sierra-moxon** [comment]
new release of biolink out with specialization_qualifiers to hold MAXO terms.

**Comment by kevinschaper** [comment]
Done!

**Comment by kevinschaper** [comment]
Not done! I need to make the associations visible, I'll keep this issue for that

**Comment by kevinschaper** [comment]
Oh, closing after all, we had #919 as the issue for the UI part.
